test/onnx/model_defs/emb_seq.py

```diff
@@ -0,0 +1,31 @@
+
+import torch.nn as nn
+
+
+class EmbeddingNetwork1(nn.Module):
+    def __init__(self, dim=5):
+        super(EmbeddingNetwork1, self).__init__()
+        self.emb = nn.Embedding(10, dim)
+        self.lin1 = nn.Linear(dim, 1)
+        self.seq = nn.Sequential(
+            self.emb,
+            self.lin1,
+        )
+
+    def forward(self, input):
+        return self.seq(input)
+
+
+class EmbeddingNetwork2(nn.Module):
```

Comment:
Looks like EmbeddingNetwork2 is a superset of EmbeddingNetwork1, shall we just keep EmbeddingNetwork2?

Reply:
We could transform it into a single network using flags

Reply:
I think Lu's point is that `EmbeddingNetwork2` is embedding->linear->sigmoid, while `EmbeddingNetwork1` is just embedding->linear, so why not just test `EmbeddingNetwork2` since if `EmbeddingNetwork1` fails won't `EmbeddingNetwork2` always fail?